src/map/clif.c

```diff
@@ -9851,21 +9851,37 @@ static int clif_guess_PacketVer(int fd, int get_previous, int *error)
 		err = 0;\
 		if( error )\
 			*error = 0;\
+		if( diskey ) ShowWarning("Client use no encryption while server do, please fix me\n"); \
```

Comment:
OK, is only 1 message per connection attempt but you're right.
1 question tough, should I keep it with a define set as off by default or completly remove it ?
(I'd prefer the define option if you ask me)